test/testgroups/operatorsourcetesgs.go

```diff
@@ -0,0 +1,37 @@
+package testgroups
+
+import (
+	"testing"
+
+	"github.com/operator-framework/operator-marketplace/test/helpers"
+	"github.com/operator-framework/operator-marketplace/test/testsuites"
+	"github.com/operator-framework/operator-sdk/pkg/test"
+)
+
+// OperatorSourceTestGroup creates an OperatorSource and then runs a series of test suites that rely on this resource.
+func OperatorSourceTestGroup(t *testing.T) {
```

Comment:
Please re-factor as per @kevinrizza's suggestion

Reply:
Will do!

Reply:
Resolved in the latest commit.